Index: Function.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>a = 5\r\nb = 10\r\n\r\n\r\ndef test_function(num_1, num_2):\r\n    result = (num_1 * num_2)\r\n    return result\r\n\r\n\r\nprint(test_function(a, b))\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Function.py b/Function.py
--- a/Function.py	(revision e81291fcf2207c9125ffd2f01c4f9e0d3b7b4668)
+++ b/Function.py	(date 1666095822993)
@@ -7,4 +7,4 @@
     return result
 
 
-print(test_function(a, b))
+print(test_function(a, b))
\ No newline at end of file
Index: .idea/discord.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/discord.xml b/.idea/discord.xml
new file mode 100644
--- /dev/null	(date 1666088757893)
+++ b/.idea/discord.xml	(date 1666088757893)
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="DiscordProjectSettings">
+    <option name="show" value="PROJECT_FILES" />
+    <option name="description" value="" />
+  </component>
+</project>
\ No newline at end of file
